responsibleai/tests/counterfactual/test_counterfactual_advanced_features.py

```diff
@@ -153,6 +154,87 @@ def test_counterfactual_manager_save_load(self, tmpdir):
         assert counterfactual_config_list[0].explainer is not None
         assert counterfactual_config_list[1].explainer is not None
 
+    @pytest.mark.parametrize('feature_importance', [True, False])
+    def test_counterfactual_manager_request_counterfactuals(
+            self, feature_importance):
+        X_train, X_test, y_train, y_test, feature_names, _ = \
+            create_iris_data()
+
+        model = create_lightgbm_classifier(X_train, y_train)
+        X_train['target'] = y_train
+        X_test['target'] = y_test
+
+        rai_insights = RAIInsights(
+            model=model,
+            train=X_train,
+            test=X_test.iloc[0:10],
+            target_column='target',
+            task_type='classification')
+
+        rai_insights.counterfactual.add(
+            total_CFs=10, desired_class=2,
+            features_to_vary=[feature_names[0]],
+            permitted_range={feature_names[0]: [2.0, 5.0]},
+            feature_importance=feature_importance)
+        rai_insights.counterfactual.add(
```

Comment:
nit: maybe put this in a loop? it looks like same values are used except for desired class, so you can loop through desired_class = [1, 2] perhaps.

Reply:
I guess the configs aren't too many at this point. Maybe we can do the suggested way once we have more such configs.